game/gamestates/gameover.lua

```diff
@@ -0,0 +1,26 @@
+
+local PlayerDeathView = require 'view.playerdeath'
+
+local state = {}
+
+function state:enter(from, player)
+  self.view = PlayerDeathView(player)
+  self.view:addElement("HUD")
```

Comment:
I think state functions are not called as methods, and `state.enter()` actually receives the previous state as its first parameter, so this code does something weird that ends up working in the end.

Reply:
[https://hump.readthedocs.io/en/latest/gamestate.html](nope)

Reply:
Oh yes,  you are right. I got confused because they are used like objects, but we create them like modules. Also, why dat url.